spec/helpers/time_options_helper_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe TimeOptionsHelper do
```

Comment:
Test class is great! Can you squash it up to the same commit as `TimeOptionsHelper`.

Reply:
Already done 👍 can I give you a nudge when this is ready to re-review? There's no way to put a GitHub PR back into Draft and I'm still pushing up changes that aren't ready to view yet.